docs/streamdetails.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Stream Details
+description: A Description of the Stream Details View
+---
+
+# Stream Details
+
+## Basic View
+![image](assets/screenshots/streamdetails-basic.png)
+
+This view shows the complete path that the stream travels. A blue dot on the line on the left shows that a modification has been made to the stream at that point.
```

Comment:
```suggestion
This view shows the complete path that the stream travels. A blue dot on the line on the left shows that a (lossless) modification has been made to the stream at that point.
```

Reply:
The blue dot is also used on the MP3 line so Maxim would have to change that…?

Reply:
ah that blue/green-ish dot you mean... yeah that is basically just a bullet to indicate one step in the process.
Maybe rephrase your text to represnet just that: Each dot represents one step in the signal path.